### PR TITLE
Api caller

### PR DESCRIPTION
**Pull request by AndrewCushing**
Search functionality done. Upcoming films now pulls from back end. Current listings pulls from back end but data still needs to be added to db. Film details page for each film now pulls from back end but needs work to display full film details (will either add a flag for which page it is on or make a new component).

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
 
 # dependencies
-/node_modules
+node_modules
```
Comment (ianowenQA) [inline .gitignore]: why was this changed?

File: nbs-cinema/src/component/CurrentFilms.js [hunk 2]
```diff
@@ -1,28 +1,38 @@
 import React, { Component } from 'react';
 import MovieRow from './MovieRow.js'
 
+import { BrowserRouter as Router} from 'react-router-dom'
```
Comment (ianowenQA) [inline nbs-cinema/src/component/CurrentFilms.js]: Semicolons!

File: nbs-cinema/src/component/FilmDetails.js [hunk 3]
```diff
@@ -1,74 +1,60 @@
 import React, { Component } from 'react';
-import { Link } from 'react-router-dom';
+import MovieRow from './MovieRow.js'
```
Comment (ianowenQA) [inline nbs-cinema/src/component/FilmDetails.js]: Semicolons!

File: nbs-cinema/src/component/HeaderComponent.js [hunk 4]
```diff
@@ -23,22 +37,23 @@ class HeaderComponent extends Component {
...
-                    </ul>
-                        </div>
+                    <div>
+                        <ul className="navbar-nav">
+                            <li><Link className="nav-link" to="/Screens">Screens</Link></li>
+                            <li><Link className="nav-link" to="/CurrentFilms">Listings</Link></li>
+                            <li><Link className="nav-link" to="/new">New Releases</Link></li>
+                        </ul>
+                    </div>
                     <div style={{marginLeft:400,paddingTop:11}}className="input-group mb-4 fixed-right">
```
Comment (ianowenQA) [inline nbs-cinema/src/component/HeaderComponent.js]: Consider moving these inline styles to a CSS class.

File: nbs-cinema/src/component/SearchResults.js [hunk 5]
```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react';
+import MovieRow from './MovieRow.js'
+
+import { BrowserRouter as Router} from 'react-router-dom'
```
Comment (ianowenQA) [inline nbs-cinema/src/component/SearchResults.js]: Semicolons!

File: .gitignore (hunk 1 above)
Comment (AndrewCushing) [reply .gitignore]: Because the node modules weren't being ignored. After removing the slash they were getting ignored. Is that not right?

File: nbs-cinema/src/component/HeaderComponent.js (hunk 4 above)
Comment (AndrewCushing) [reply nbs-cinema/src/component/HeaderComponent.js]: I'll make a note for a future change. It wasn't the purpose of this branch though
